applicationinsights.ts

```diff
@@ -307,6 +318,33 @@ export class Configuration {
         return Configuration;
     }
 
+    /**
+     * Sets the state of Web snippet injection
+     * @param value if true Web snippet will be tried to be injected in server response
+     * @returns {Configuration} this class
+     */
+    public static setWebSnippetInjection(value: boolean) {
```

Comment:
Is cleaner to have a single method to enable this and extra parameter to determine if debug version or not